package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A JavaScript plugin to warn users about links to private pages.",
   "main": "private-eye.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
```

Comment:
Mind adding a note to the README or CONTRIBUTING files about how to run the tests?

Reply:
Done - added to README.md, let me know if there's a better way to phrase it/place, it feels a little tacked on